### ISSUE TITLE
Higher Resolution Pre-training

### ISSUE THREAD

**Issue by ajamal-isrg** [naver/unic#4]
Hi, 

Thanks for sharing the great work open source. I have a question regarding pre-training at resolution different than 224px. You guys shared the model with 224/336 for ViT-L student. I was wondering, was the model trained with 224px first and then in the second stage, it was trained with 336px for small amount of iterations? or was it pre-trained directly with 336px?

**Comment by mbsariyildiz** [comment]
Hi,

Thanks for your interest!
That model indeed had a two-stage training, first with resolution of 224, and then 336, please see Table C in the supplementary (pasting here for convenience)

<img width="2640" height="996" alt="Image" src="https://github.com/user-attachments/assets/7e33c2de-d2cb-4e9a-aa81-6ac2f35643c8" />

**Comment by ajamal-isrg** [comment]
Thanks for sharing this.
For the second stage, did you use the same normal setup for hyper-params such as lr=3e-4, batchsize of 512 and wd=3e-2 and so on. ?

**Comment by mbsariyildiz** [comment]
I cannot find the exact lr/wd/batch-size values right now.
In the second stage, a rule of thumb was to first maximize batch size for a A/H100 with 80GB of memory, and using a lower effective learning rate compared to the first stage. We kept wd the same.
Hope that helps.

**Comment by ajamal-isrg** [comment]
Another question regarding the positional embedding in the second stage? Did you interpolate the positional embedding of stage-1 checkpoint and loaded them before kickstart the stage-2 of the distillation or did you guys use the same positional embedding of 224px but still trained it with 336px by interpolating it during every forward pass?

**Comment by mbsariyildiz** [comment]
We tried these two options and saw no significant impact. What remained in the code was that we restart positional embeddings from scratch for fine-tuning.

However, in the follow-up work, [DUNE](https://github.com/naver/dune), resizing positional embeddings for the fine-tuning stage helped, [see here for code](https://github.com/naver/dune/blob/main/utils/exp.py#L170)

Hope that helps.

**Comment by ajamal-isrg** [comment]
Thanks for the information, I started stage-2 fine-tuning by resizing positional embeddings but looks like, training it for another 100 epochs degrades the downstream performance. Only training for 10 epochs in stage-2 yields the highest performance.
FYI, downstream dataset is different from what is reported in the paper.

**Comment by mbsariyildiz** [comment]
Thanks for sharing your experience. Good luck with your experiments. Closing the issue for now, feel free to re-open if needed.
